docs/tutorials/04-include-in-your-build-tool.md

```diff
@@ -112,64 +112,64 @@ This code ensures that your application is re executed in an isolated environmen
 package main
 
 import (
-	"context"
-	"fmt"
-	"github.com/containers/buildah"
-	"github.com/containers/buildah/define"
-	"github.com/containers/storage/pkg/unshare"
-	is "github.com/containers/image/v5/storage"
-	"github.com/containers/image/v5/types"
-	"github.com/containers/storage"
+  "context"
+  "fmt"
+  "github.com/containers/buildah"
+  "github.com/containers/buildah/define"
+  "github.com/containers/storage/pkg/unshare"
+  is "github.com/containers/image/v5/storage"
+  "github.com/containers/image/v5/types"
+  "github.com/containers/storage"
 )
 
 func main() {
-	if buildah.InitReexec() {
-		return
-	}
-	unshare.MaybeReexecUsingUserNamespace(false)
+  if buildah.InitReexec() {
+    return
+  }
+  unshare.MaybeReexecUsingUserNamespace(false)
 
-	buildStoreOptions, err := storage.DefaultStoreOptions(unshare.IsRootless(), unshare.GetRootlessUID())
+  buildStoreOptions, err := storage.DefaultStoreOptions(unshare.IsRootless(), unshare.GetRootlessUID())
 
-	if err != nil {
-		panic(err)
-	}
+  if err != nil {
+    panic(err)
+  }
 
-	buildStore, err := storage.GetStore(buildStoreOptions)
+  buildStore, err := storage.GetStore(buildStoreOptions)
 
-	if err != nil {
-		panic(err)
-	}
+  if err != nil {
+    panic(err)
+  }
 
-	opts := buildah.BuilderOptions{
-		FromImage:        "node:12-alpine",
-		Isolation:        define.IsolationChroot,
-		CommonBuildOpts:  &define.CommonBuildOptions{},
-		ConfigureNetwork: define.NetworkDefault,
-		SystemContext: 	  &types.SystemContext {},
-	}
+  opts := buildah.BuilderOptions{
+    FromImage:        "node:12-alpine",
+    Isolation:        define.IsolationChroot,
+    CommonBuildOpts:  &define.CommonBuildOptions{},
+    ConfigureNetwork: define.NetworkDefault,
+    SystemContext:    &types.SystemContext {},
+  }
 
-	builder, err := buildah.NewBuilder(context.TODO(), buildStore, opts)
+  builder, err := buildah.NewBuilder(context.TODO(), buildStore, opts)
 
-	if err != nil {
-		panic(err)
-	}
+  if err != nil {
+    panic(err)
+  }
 
-	err = builder.Add("/home/node/", false, buildah.AddAndCopyOptions{}, "script.js")
+  err = builder.Add("/home/node/", false, buildah.AddAndCopyOptions{}, "script.js")
 
-	if err != nil {
-		panic(err)
-	}
+  if err != nil {
+    panic(err)
+  }
 
-	builder.SetCmd([]string{"node", "/home/node/script.js"})
+  builder.SetCmd([]string{"node", "/home/node/script.js"})
 
-	imageRef, err := is.Transport.ParseStoreReference(buildStore, "docker.io/myusername/my-image")
+  imageRef, err := is.Transport.ParseStoreReference(buildStore, "docker.io/myusername/my-image")
 
-	if err != nil {
-		panic(err)
-	}
+  if err != nil {
+    panic(err)
+  }
 
-	imageId, _, _, err := builder.Commit(context.TODO(), imageRef, define.CommitOptions{})
+  imageId, _, _, err := builder.Commit(context.TODO(), imageRef, buildah.CommitOptions{})
```

Comment:
We want to encourage people to use define rather then buildah, especially for things like remote commands in podman-remote.

Reply:
I don't have the entire context of the interaction between this and external implementations. First time using Buildah here and I have no intentions to use Podman in the near future.
What I can tell is that running the example code from this tutorial as it is, does not compile and the only place I found the definition of ```CommitOptions``` is under buildah. It seems I'm missing something here. Can you help me understand what may be?

Reply:
Oops you are right, I misread this as BuildOptions.

LGTM